packages/caliper-tests-integration/fabric_tests/caliper.yaml

```diff
@@ -42,7 +42,7 @@ caliper:
                 target: console
                 enabled: true
                 options:
-                    level: info
+                    level: debug
```

Comment:
can result in travis log size being exceeded

Reply:
You're right, I was just checking the usefulness of debug messages, will change it back

Reply:
Changed back.